packages/main/src/components/add-prefix-modal/utils/Common.jsx

```diff
@@ -0,0 +1,120 @@
+import { authAxios } from "./Api";
+import styles from "./styles/PaymentMethod.module.css";
+import mastercard from "../assets/Mastercard.svg";
+import jcb from "../assets/jcb.svg";
+import visa from "../assets/visa.svg";
+import amex from "../assets/american-express.svg";
+import dinner from "../assets/dinners-club.svg";
+import discover from "../assets/discover.svg";
+
+// return the token from the session storage
```

Comment:
use jsdocs when commenting functions so that your IDE actually shows it
```javascript
/*
 * Returns the token from the session storage
 */
const getToken = () => void
```